packages/paste-website/src/components/homepage/HomeHero.tsx

```diff
@@ -66,14 +66,14 @@ const HomeHero: React.FC = () => {
             <Column span={5}>
               <NewComponentBanner>
                 <NewComponentBannerBadge>New!</NewComponentBannerBadge>
-                <NewComponentBannerText>We just added the Date Picker!</NewComponentBannerText>
+                <NewComponentBannerText>Did you know we&apos;ve got an Alert Dialog?!</NewComponentBannerText>
```

Comment:
You may need to rebase now that you merged this change in another PR. I don't think you intended to make this change in this PR

Reply:
Yeah, I mentioned it at the top, twas a mistake- this change is merged in now so hopefully it won't be too much of a problem